### ISSUE TITLE
Ping to google

### ISSUE THREAD

**Issue by fakirsu** [CharlieEtienne/t2s#55]
Hello, 
I have the feeling that :
- If I click on "Listen" first, the tool asks Google
- If then I click on Download, I have the impression that it queries Google again, then it downloads.

Am I wrong ? 
If so, it's a shame, because it doubles the consumption of the Google API.

Also, I noticed something that isn't quite a bug. Breeding:
- I have 3 texts/recordings to produce.
- For the first, I first click on "Listen", then on "Download"
- For the second, I only click on "Download"
- For the third, if this time I want to listen using the playback bar (the one above the "listen" button), the tool plays me recording n° 1.
In other words, in the playback bar, the tool remembers the last recording that was produced by the "listen" button, but not really the last one that was requested from Google.
